keg_apps/web/views/routing.py

```diff
@@ -60,10 +60,10 @@ def get(self):
 
 class HelloWorld(BaseView):
     """
-        /hello -> 'Hello World'
-        /hello/foo -> 'Hello Foo'
+        /hello-world -> 'Hello World'
+        /hello-world/foo -> 'Hello Foo'
     """
-    # relative URL indicates this route should be appended to the default rule for the class
+    rule('')
```

Comment:
Am I misunderstanding, or does the readme above indicate that this should be `rule()`?

Reply:
Both will work, but `rule()` would be better IMO. I'll update it here.